test/stock-order.spec.js

```diff
@@ -0,0 +1,163 @@
+const { 
+  spawnSync, 
+  //fork
+} = require('child_process');
+const path = require('path');
+const fs = require('fs-extra');
+const ExcelJS = require('exceljs');
+
+const { stockOrderScenario } = require('./mocks/mocks');
+const { 
+  setDirToprojectConfig,
+  revertBackToProjectHome
+} = require('./test-utils');
+
+
+describe('Stock order', () => {
+  const workingDir = process.cwd();
+
+  beforeEach(() => {
+    revertBackToProjectHome(workingDir);
+    setDirToprojectConfig();
+  });
+
+  afterEach(() => {
+    revertBackToProjectHome(workingDir);
+  });
+
+  it('Add stock order summaries test', async() => {
```

Comment:
@ernestoteo , These are like integration tests. I think the unit test will focus on testing small unit like functions